src/pkg/cli/new_test.go

```diff
@@ -0,0 +1,16 @@
+package cli
+
+import (
+	"context"
+	"testing"
+)
+
+func TestInitFromSamples(t *testing.T) {
+	err := InitFromSamples(context.Background(), t.TempDir(), []string{"nonexisting"})
+	if err == nil {
+		t.Fatal("Expected test to fail")
+	}
+	if err.Error() != "sample not found" {
```

Comment:
you can actually improve this test now by using `errors.Is(err, …)` to ensure it returns the new var